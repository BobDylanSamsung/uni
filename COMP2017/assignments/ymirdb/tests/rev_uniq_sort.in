set a 1 2 3 4 5
get a
rev a
get a
sort a
get a
push a 1 2 3 4 5
uniq a
get a
uniq a
get a
sort a
get a
uniq a
set b 2 4 1 -5 2 -1 7 -2 1 2 -1 0 0 5 -5
list entries
rev b
get b
rev b
get b
sort b
get b
uniq b
get b
uniq b
list entries
bye
