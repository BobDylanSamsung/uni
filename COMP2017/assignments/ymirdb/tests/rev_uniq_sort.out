ok

[1 2 3 4 5]

ok

[5 4 3 2 1]

ok

[1 2 3 4 5]

ok

ok

[5 4 3 2 1 2 3 4 5]

ok

[5 4 3 2 1 2 3 4 5]

ok

[1 2 2 3 3 4 4 5 5]

ok

ok

b [2 4 1 -5 2 -1 7 -2 1 2 -1 0 0 5 -5]
a [1 2 3 4 5]

ok

[-5 5 0 0 -1 2 1 -2 7 -1 2 -5 1 4 2]

ok

[2 4 1 -5 2 -1 7 -2 1 2 -1 0 0 5 -5]

ok

[-5 -5 -2 -1 -1 0 0 1 1 2 2 2 4 5 7]

ok

[-5 -2 -1 0 1 2 4 5 7]

ok

b [-5 -2 -1 0 1 2 4 5 7]
a [1 2 3 4 5]

bye
